darshan-util/pydarshan/pyproject.toml

```diff
@@ -3,3 +3,40 @@ requires = [
     "wheel",
     "setuptools",
 ]
+
+[tool.cibuildwheel]
+environment = "PYDARSHAN_BUILD_EXT=1"
+build = "pp39*"
+skip = [
+    "*musllinux*",
+    "*i686*"
+]
+test-requires = [
+    "pytest",
+    "lxml",
+    "matplotlib<3.5",
+    "importlib_resources;python_version<'3.9'"
+]
+test-command = "pytest {package}"
+
+[tool.cibuildwheel.linux]
```

Comment:
From the example: https://cibuildwheel.readthedocs.io/en/stable/options/#configuration-file
```
[tool.cibuildwheel.linux]
repair-wheel-command = "auditwheel repair -w {dest_dir} {wheel}"
```

And more info here:
https://cibuildwheel.readthedocs.io/en/stable/options/#repair-wheel-command

Reply:
Thanks for catching that, I'll have a look at this and below and try to add them to the pyprojec.toml